mm-mock-server/src/mocks/util/generators.ts

```diff
@@ -113,15 +116,15 @@ export function generateChannelInvitation(channelParticipants: any[], declined?:
         __typename: "ChannelInvitation",
         id: faker.string.alphanumeric({length: 24}),
         channelName: faker.lorem.words(2),
-        channel: generateChannel(channelParticipants),
+        channel: channel,
```

Comment:
You could shorten this to `channel,`, just like `recipient,` in line 125 or `sender,` in line 127